projects/wave-core-new/src/lib/operators/dialogs/operator-list/operator-list.component.ts

```diff
@@ -207,35 +211,35 @@ export class OperatorListComponent implements OnInit {
             this.searchString$.pipe(map(s => s.toLowerCase())),
         ]).pipe(
             map(([operatorGroups, searchString]) => {
-                    const nameComparator = (a: string, b: string): number => {
-                        const stripped = (s: string): string => s.replace(' ', '');
-
-                        return stripped(a).localeCompare(stripped(b));
-                    };
+                const nameComparator = (a: string, b: string): number => {
+                    const stripped = (s: string): string => s.replace(' ', '');
 
-                    const filteredGroups = [];
-                    for (const group of operatorGroups) {
-                        const operators = [];
-                        for (const operator of group.list) {
-                            const searchMatchesTypeName = () => operator.type.NAME.toLowerCase().includes(searchString);
-                            const searchMatchesDescription = () => operator.description.toLowerCase().includes(searchString);
+                    return stripped(a).localeCompare(stripped(b));
+                };
 
+                const filteredGroups = [];
+                for (const group of operatorGroups) {
+                    const operators = [];
+                    for (const operator of group.list) {
+                        const searchMatchesTypeName = () => operator.type.NAME.toLowerCase().includes(searchString);
+                        const searchMatchesDescription = () => operator.description.toLowerCase().includes(searchString);
 
-                            if (searchMatchesTypeName() || searchMatchesDescription()) {
-                                operators.push(operator);
-                            }
-                        }
 
-                        if (operators.length > 0) {
-                            filteredGroups.push({
-                                name: group.name,
-                                list: operators.sort((a, b) => nameComparator(a.type.NAME, b.type.NAME)),
-                            });
+                        if (searchMatchesTypeName() || searchMatchesDescription()) {
+                            operators.push(operator);
                         }
                     }
 
-                    return filteredGroups.sort((a, b) => nameComparator(a.name, b.name));
+                    if (operators.length > 0) {
+                        filteredGroups.push({
+                            name: group.name,
+                            list: operators.sort((a, b) => nameComparator(a.type.NAME, b.type.NAME)),
+                        });
+                    }
                 }
+
+                return filteredGroups.sort((a, b) => nameComparator(a.name, b.name));
+            }
```

Comment:
did you change anything?

Reply:
yes, I added the operator button... format reversal: 009ab0a4c7897d1b68e9ee7e18fc7b6e600a9268